bin/dev

```diff
@@ -1,8 +1,2 @@
-#!/usr/bin/env sh
-
-if ! gem list foreman -i --silent; then
-  echo "Installing foreman..."
-  gem install foreman
-fi
-
-exec foreman start -f Procfile.dev "$@"
+#!/usr/bin/env ruby
+exec "./bin/rails", "server", *ARGV
```

Comment:
I think we should keep forman here ?

Reply:
Hmm yeah. Not sure why this changed.

Reply:
I think I saw an issue on that not sure why this was changed

Reply:
https://github.com/rails/rails/issues/53496